### PR TITLE
28_refactor_views

### PR DESCRIPTION
**Pull request by s-espinosa**
Update styling for:
* update nav
* secondary nav
* home
* coder lists

Proceeding to work remaining views.

### CODE REVIEW

**Comment by s-espinosa** [comment]
@thePaulista @thompickett Updated remaining styling. Please confirm that I should move forward and merge master into this branch so that I can resolve any conflicts to facilitate merge to master.

**Comment by s-espinosa** [comment]
@thompickett @thePaulista Pulled master and merged into my branch. No conflicts. Passing test. Recommend we merge this branch to master to update views.

File: app/views/coders/_attributes.html.erb [hunk 1]
```diff
@@ -1 +1 @@
-<span class="center-align"><%= button_to "Add Genius", teams_path(coder_id: coder.id)%></span>
```
Comment (thompickett) [inline app/views/coders/_attributes.html.erb]: we might want to make the button names consistent

**Comment by thompickett** [comment]
@s-espinosa Looks great! Good Job.

**Pull request merged**